product.go

```diff
@@ -27,6 +27,8 @@ type ProductParams struct {
 	Name              string             `form:"name"`
 	PackageDimensions *PackageDimensions `form:"package_dimensions"`
 	Shippable         *bool              `form:"shippable"`
+	Statement         string             `form:"statement_descriptor"`
+	Type              string             `form:"type"`
```

Comment:
`Type` should likely be using const instead of just a `string`. Example: on Account's `Type` [here](https://github.com/stripe/stripe-go/blob/master/account.go#L214-L223)